src/assets/js/main.js

```diff
@@ -62,41 +62,37 @@ async function generateChord(jsonData, year, week) {
 (async () => {
   // load data and select components
   const jsonData = await d3.json('data/flights_countries.json');
-  const yearselect = document.getElementById('year');
   const weekslider = document.getElementById('weekslider');
   const weekindicator = document.getElementById('weekindicator-week');
+  const yearindicator = document.getElementById('yearindicator');
```

Comment:
```suggestion
  const weekslider = document.querySelector('[data-js-weekslider]');
  const weekindicator = document.querySelector('[data-js-weekindicator-week]');
  const yearindicator = document.querySelector('[data-js-yearindicator]');
```

Reply:
lol das ist falsch

Reply:
querySelector('[data-js-yearindicator]');
```suggestion
  const weekslider = document.querySelector('[data-js="weekslider"');
```

etc